src/app/excel/excel-table/excel-table.component.html

```diff
@@ -0,0 +1,44 @@
+﻿<div>
+    <div class="ui-g" id="availability-table">
+        <div class="ui-g-8">
+            <h5>Dostępność - {{sprint}}</h5>
+            <p-table [value]="workers">
+                <ng-template pTemplate="header">
+                    <th>Lp</th>
+                    <th>Osoba</th>
+                    <th>Ilość zadeklarowanych godzin</th>
+                </ng-template>
+                <ng-template pTemplate="body" let-worker>
+                    <tr>
+                        <td>{{worker.id}}</td>
+                        <td>{{worker.name}}</td>
+                        <td>{{worker.declaredHours}}</td>
+                    </tr>
+                </ng-template>
+            </p-table>
+            <hr>
+            <p>W tabeli docelowo znajdzie się efekt zapytania "GET /availability"</p>
```

Comment:
mock

Reply:
A ten GET to będzie na  `/projects/{id}/sprints/{id}/availability` w sumie

Reply:
Tutaj będzie GET na `/projects/{id}/sprints/{id}/summary` - pinguj @MrMarosis